src/main/java/redstone/multimeter/server/MultimeterServer.java

```diff
@@ -69,7 +69,7 @@ public boolean isDedicatedServer() {
 	}
 
 	public File getConfigDirectory() {
-		return new File(server.getServerDirectory(), RedstoneMultimeterMod.CONFIG_PATH);
+		return new File(server.getServerDirectory().toString(), RedstoneMultimeterMod.CONFIG_PATH);
```

Comment:
Does `MinecraftServer#getServerDirectory` return a `Path` now or something? I might just move all my file interactions to NIO... but that's something for later.